source/classes.tex

```diff
@@ -1387,12 +1387,11 @@
 
 \rSec2[class.nest]{Nested class declarations}%
 \indextext{definition!nested class}%
-\indextext{class local|see{local class}}%
-\indextext{class nested|see{nested class}}
+\indextext{class!nested|see{nested class}}
 
```

Comment:
In general, it seems to me we gravitate towards "class!nested" as the main index entry (where all the page numbers go) and have "nested class" as the "see ..." entry.